draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -272,13 +272,23 @@ and using spare capacity is discussed in Sections 6, 11.1, and 12
 of {{?RFC3649}}.
 
 (2)
+: Wireless links
+
+: While the early internet was dominated by wired links, the properties
+of wireless links have become extremely important to internet performance.
+In particular, congestion controllers should be evaluated in situations
+where some packet losses are due to radio effects, rather than router
+queue drops; the bandwidth varies over time due to changing link conditions;
```

Comment:
/bandwidth/ here is used from the computing perspective but talks about radio - where this means something subtly different - should we say capacity?

Reply:
+1 about not using bandwidth. Bandwidth in radio means the width of the frequency band that carries the modulation, and the link to data rate depends on noise level (Shannon law) and number of antenna pairs (MIMO). Capacity is less ambiguous. For any given link conditions there is a tradeoff between data rate and bit error rate, but I would still use data rate rather than capacity.